EIPS/eip-4844.md

```diff
@@ -234,6 +232,41 @@ def get_origin(tx: SignedBlobTransaction) -> Address:
     return ecrecover(tx_hash(tx), int(sig.y_parity)+27, sig.r, sig.s)
 ```
 
+### Header extension
+
+The current header encoding is extended with a new 256-bit unsigned integer field `blob_txs`. This is the running
+total of excess blob txs included on chain since this EIP was activated. The resulting RLP encoding of the header is therefore:
+
+```
+rlp([
+    parent_hash,
+    ommers_hash,
+    coinbase,
+    state_root,
+    tx_root,
+    receipt_root,
+    bloom,
+    difficulty,
+    number,
+    gas_limit,
+    gas_used,
+    time,
+    extra,
+    mix_digest,
+    nonce,
+    base_fee,
+    blob_txs
+])
+```
+
+The value of `blob_txs` can be calculated using the parent header and number of blob transactions in the block.
+
+```python
+def calc_excess_blob_txs(parent: Header, blobs: int) -> int:
+    adjusted = parent.blob_txs + blobs
+    return adjusted - min(TARGET_BLOB_TXS_PER_BLOCK, adjusted)
```

Comment:
Is `TARGET_BLOB_TXS_PER_BLOCK` the same as `TARGET_BLOBS_PER_BLOCK` constant? If that's the case, can we define only one?